### PR TITLE
Redo our deprecation notice, notify about upcoming 202 deprecation

### PR DESCRIPTION
**Pull request by abrooksv**
## Screenshots (if appropriate)
Screen Shot 2021-07-08 at 11 32 19 AM
Screen Shot 2021-07-08 at 11 32 24 AM

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .changes/next-release/deprecation-8a9cd1d2-9051-4e01-b94d-c0f52687f411.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "deprecation",
+  "description" : "An upcoming release will remove support for the 2020.2 family of IDEs"
```
Comment (kiiadi) [inline .changes/next-release/deprecation-8a9cd1d2-9051-4e01-b94d-c0f52687f411.json:3]: is "family" the right word?

Comment (abrooksv) [reply .changes/next-release/deprecation-8a9cd1d2-9051-4e01-b94d-c0f52687f411.json:3]: Not sure,

Maybe "for IDEs based on 2020.2"?

File: resources/resources/software/aws/toolkits/resources/MessagesBundle.properties [hunk 2]
```diff
@@ -110,6 +110,8 @@ aws.terminal.action=Open AWS Local Terminal
 aws.terminal.action.tooltip=Start a local terminal with the current AWS connection settings injected as environment variables
 aws.terminal.exception.failed_to_resolve_credentials=Unable to open AWS-aware Terminal, unable to resolve credentials: {0}
 aws.terminal.exception.invalid_credentials=Unable to open AWS-aware Terminal, credentials aren't valid: {0}
+aws.toolkit_deprecation.title=AWS Toolkit deprecation notice
+aws.toolkit_deprecation.message=Support for {0} {1} is being deprecated - an upcoming release will require a version based on {2} or newer.
```
Comment (kiiadi) [inline resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:114]: I reckon you could do:

`Support for {0} {1} is being deprecated - an upcoming release will require {0} {2} or newer.`

Which (presumably) would translate into :

"Support for IntelliJ IDEA 2020.1 is being deprecated - an upcoming release will require IntelliJ IDEA 2020.2 or newer."

Comment (kiiadi) [reply resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:114]: but whatever - we're nitpicking at this point - the wording looks good.

**Review by kiiadi — APPROVED** [review]

**Review by rli — APPROVED** [review]

**Pull request merged**